benchmark/diagnostic_counter.h

```diff
@@ -15,15 +15,23 @@
 #ifndef HLASMPLUGIN_LANGUAGESERVER_DIAGNOSTIC_COUNTER_H
 #define HLASMPLUGIN_LANGUAGESERVER_DIAGNOSTIC_COUNTER_H
 
+#include <string>
+#include <unordered_map>
+#include <unordered_set>
+#include <utility>
+#include <vector>
+
 #include "nlohmann/json.hpp"
+#include "protocol.h"
 #include "workspace_manager.h"
 
 namespace hlasm_plugin::benchmark {
 
 class diagnostic_counter : public hlasm_plugin::parser_library::diagnostics_consumer
 {
 public:
-    void consume_diagnostics(hlasm_plugin::parser_library::diagnostic_list diagnostics) override
+    void consume_diagnostics(hlasm_plugin::parser_library::diagnostic_list diagnostics,
```

Comment:
any idea on possible reason why we would want to collect and report the fade stats in benchmark?